src/main/cs/RLBotDotNet/RLBotDotNet/GameState/DesiredVector3.cs

```diff
@@ -0,0 +1,42 @@
+﻿using rlbot.flat;
+using FlatBuffers;
+
+namespace RLBotDotNet.GameState
+{
+    public class DesiredVector3
```

Comment:
Please add a constructor that takes zero arguments. Currently the only way to make the ball drift at Z = 300 is to construct with x, y, and z, then go back and set x and y to null.

Also, would you mind adding some test code to TestBot?